README.md

```diff
@@ -35,25 +34,44 @@ You need [Google Chrome](https://www.google.com/chrome/browser/desktop/). *not t
 
 <img src="https://github.com/isaacbernat/netflix-to-srt/blob/master/chrome_console.png?raw=true" alt="Chrome console screenshot" width="557px" height="607px">
 
-### From Netflix: method 3
+## Method 3
 The information is extracted from [this post](http://forum.opensubtitles.org/viewtopic.php?t=15141).
 
 You need FireFox and AdblockPlus Add-On. *not tested on other browsers*
 - Start Netflix and your movie/episode (stream is active!)
 - Start AdblockPlus, open blockable items
-- Search: dfxp *(e.g. >> #.nflximg.com/#/#/########.dfxp?v=1&e=#########&t=######_#####&random=1234567890)*
+- Search: dfxp *(e.g. `>> #.nflximg.com/#/#/########.dfxp?v=1&e=#########&t=######_#####&random=1234567890`)*
 - open the dfxp in a new window
 - Save as
 
-### From YouTube
+# 2. YouTube-To-SRT
 - Install [youtube-dl](https://github.com/ytdl-org/youtube-dl) (available for Windows, Mac and Linux)
 - Download subs from the YouTube URL you like e.g. `youtube-dl --all-subs "https://www.youtube.com/watch?v=VHNfvFOBC0A"`
 - Subtitles should be downloaded in the same folder were the command was ran. E.g. `NameOfTheVideo VHNfvFOBC0A.ca.vtt, NameOfTheVideo VHNfvFOBC0A.tlh.vtt`
 - If you are missing a language, check that it's actually available. E.g. `youtube-dl --list-subs "https://www.youtube.com/watch?v=VHNfvFOBC0A"`
+> **Note:** you can also use [youtube-dlp](https://github.com/yt-dlp/yt-dlp-wiki/blob/master/Installation.md) which is the fork version of [youtube-dl](https://github.com/ytdl-org/youtube-dl) by [Snap](https://snapcraft.io/yt-dlp)
+ - Install [yt-dlp](https://github.com/yt-dlp/yt-dlp) using Snap:
+ ```bash
+ sudo snap install --edge yt-dlp
+ ```
+ - To download subtitles in SRT format from a YouTube URL, use:
+ ```bash
+  yt-dlp --skip-download --write-auto-subs --convert-subs srt --sub-lang "en" "https://youtu.be/cVsyJvxX48A" 
+ ```
+ This command downloads only the subtitles in SRT format.
+ - If you want to download the subtitles in VTT format, you can use:
+ ```bash
+ yt-dlp --skip-download --write-auto-subs --sub-lang "en" "https://youtu.be/cVsyJvxX48A"
+ ```
+ - To download the video with audio and subtitles, simply omit the --skip-download option:
+ ```bash
+ yt-dlp --write-auto-subs --sub-lang "en" "https://youtu.be/cVsyJvxX48A"
+ ```
+> **Note:** If you don't enable subtitles in the YouTube video (captions), then the command will not work as expected. Make sure to enable subtitles before running the command.
 
-## Convert them into .srt
```

Comment:
hmm, I see. I didn’t get your meaning, but basically, I just renamed it from 'Convert them into .srt' to 'Clone Repo & Run Files,' nothing else. I did there because in yt-dlp I already added the command to install, and yt-dlp automatically downloads subtitles in .srt format. So, I thought renaming it to 'Clone Repo' would make more sense since most of the text is about cloning the repo. It will be easier for users to find what they need that way. That’s it. But I didn't get your meaning, can you clarify it if you don’t mind